### PR TITLE
[WiP] change image of ingress controller

### PR DESCRIPTION
**Pull request by linki**
We need to change the image of the ingress controller. It changed to make it more consistent.

This is a reminder. Currently CDP somehow doesn't build it.

### CODE REVIEW

File: cluster/manifests/ingress-controller/deployment.yaml [hunk 1]
```diff
@@ -26,7 +26,7 @@ spec:
         operator: Exists
       containers:
       - name: controller
-        image: registry.opensource.zalan.do/teapot/kube-aws-ingress-controller:v0.3.5
+        image: registry.opensource.zalan.do/teapot/kube-ingress-aws-controller:v0.3.5
```
Comment (szuecs) [inline cluster/manifests/ingress-controller/deployment.yaml]: the former looks more correct, not?

Comment (mikkeloscar) [reply cluster/manifests/ingress-controller/deployment.yaml]: I also think the former sounds better, but the repository is called `kube-ingress-aws-controller` so it makes sense to keep it consistent like @linki did :+1:

Comment (linki) [reply cluster/manifests/ingress-controller/deployment.yaml]: Exactly @mikkeloscar. Other option is to rename the repo but then I would go straight to `alb-ingress-controller`.

**Comment by szuecs** [comment]
:-1:

**Comment by szuecs** [comment]
Let's close this in favour of https://github.com/zalando-incubator/kubernetes-on-aws/pull/463

**Pull request closed without merging**